script/tool/test/common/git_version_finder_test.dart

```diff
@@ -88,6 +88,17 @@ file2/file2.cc
     verify(gitDir
         .runCommand(<String>['diff', '--name-only', customBaseSha, 'HEAD']));
   });
+
+  test('include uncommitted files if requested', () async {
+    const String customBaseSha = 'aklsjdcaskf12312';
+    gitDiffResponse = '''
+file1/pubspec.yaml
+file2/file2.cc
+''';
+    final GitVersionFinder finder = GitVersionFinder(gitDir, customBaseSha);
+    await finder.getChangedFiles(includeUncommitted: true);
+    verify(gitDir.runCommand(<String>['diff', '--name-only', customBaseSha]));
```

Comment:
A comment that you are looking for the absence of "HEAD" in the args would help make this more clear.

Reply:
Done.